test/cli/detailed-errors/test.sh

```diff
@@ -1,5 +1,13 @@
 #!/usr/bin/env bash
 
-set -euo pipefail
+cwd="$(pwd)"
 
-main/sorbet --silence-dev-message --max-threads=0 test/cli/detailed-errors 2>&1
+cd test/cli/detailed-errors
+
+"$cwd/main/sorbet" --silence-dev-message --max-threads=0 attached_class.rb hash.rb shapes.rb type_members.rb class_of.rb intersection.rb tuples.rb 2>&1
```

Comment:
I made this changes because classes I was declaring in `union.rb` were conflicting with classes declared in other files. I'll move each file to have its own invocation of Sorbet in a future PR.

Reply:
slight preference to not have multiple invocations, because the CLI tests that have multiple debug-mode sorbet invocations are noticeably slower to run locally, but it's not that big a deal

(my preference would be to wrap the entire file with something like `module Union` or something)